.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
```

Comment:
- [ ] It would be better to include a link to the frontend part so code reviewers can fully review it. It would be better to include the link to the frontend PR in this PR and README